### DISCUSSION TITLE
A question about testing a model

### DISCUSSION THREAD

**Discussion by sooyaaa233** [deepmodeling/deepmd-kit#2582] [Q&A]
The following error occurred when running                     predict=training_systems.predict("graph.pb") 
![image](https://github.com/deepmodeling/deepmd-kit/assets/126163538/0dead00e-f1e3-4abe-8c65-d19ac72b41b0)

**Comment by njzjz** [comment]
It seems that the elements in `training_systems` is a super set of those in `graph.pb`, or there is no element information in `graph.pb`.
